client/src/utils/addHelper.js

```diff
@@ -0,0 +1,19 @@
+export const addHelper = data => {
+  return fetch("/add", {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(data)
+  }).then(response => {
+    return response.json();
+  });
+};
+
+export const getlastkm = data => {
+  return fetch("/getstartkm", {
+    method: "GET",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify(data)
+  }).then(response => {
+    return response.json();
+  });
+};
```

Comment:
It doesn't make sense to have file name same as one of the exported function of the file itself, suggested name could be `fetch_helper` or may be creating another file for the `getlastkm` function

Reply:
Yeah maybe change filename to `requests.js` or something. Also `add` is soooo vague and specific.... `addLogs` or `addTrips`?